code/modules/multiz/movement/mob/living_zmove.dm

```diff
@@ -86,6 +86,9 @@
 
 /// Actually starts a zMove, doing movement animations
 /mob/living/proc/start_travel_z(mob/user, upwards = TRUE, move_verb = "floating", delay = 3 SECONDS, allow_movement = TRUE)
+	if(get_turf(usr) in user.do_afters)
```

Comment:
For consistency, this should be `get_turf(user)` since it uses that specifically as the target later on when running creating the `do_after` on line 99

```suggestion
	if(get_turf(user) in user.do_afters)
```